### PR TITLE
Optimize acceptance test CI

### PR DESCRIPTION
**Pull request by Ivo-Yankov**
**Description**:
The Acceptance API CI takes up to 40 minutes to complete. This PR splits it into 3 smaller chunks to reduce the execution time.

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Comment by Ivo-Yankov** [comment]
The settings of the `API / acceptance-workflow` CI need to be changed.

File: packages/server/tests/clients/servicesClient.ts [hunk 1]
```diff
@@ -286,6 +287,14 @@ export default class ServicesClient {
         return accountBalance.hbars;
     }
 
+    async getFileContent(fileId: string): Promise<any> {
+        const query = new FileContentsQuery()
+            .setFileId(fileId);
+
+        const data = await query.execute(this.client);
```
Comment (natanasow) [inline packages/server/tests/clients/servicesClient.ts:294]: nit: we can skip the variable assignment `return await query.execute(this.client)`

**Review by natanasow — APPROVED** [review]
LG

**Review by Nana-EC — CHANGES_REQUESTED** [review]
Fix CI failure - "ReferenceError: constants is not defined"

File: .github/workflows/acceptance.yml [hunk 2]
```diff
@@ -8,11 +8,23 @@ on:
     tags: [v*]
 
 jobs:
-  api:
-    name: API
+  api_chunk_1:
```
Comment (Nana-EC) [inline .github/workflows/acceptance.yml:11]: Do the chunks have any logic to the grouping?
If so let's name it appropriately.
If not maybe `api_batch_1` etc is better.

**Review by Nana-EC — CHANGES_REQUESTED** [review]
LG, just a nit rename from chunk to batch or something better.

File: .github/workflows/acceptance.yml (hunk 2 above)
Comment (Ivo-Yankov) [reply .github/workflows/acceptance.yml]: There isn't any speciffic logic to the grouping. I changed it from `chunk` to `batch`.

File: .github/workflows/acceptance.yml [hunk 3]
```diff
@@ -8,11 +8,23 @@ on:
     tags: [v*]
 
 jobs:
-  api:
-    name: API
+  api_batch_1:
+    name: API Chunk 1
```
Comment (Nana-EC) [inline .github/workflows/acceptance.yml:12]: nit: here and 2 below
```suggestion
    name: API Batch 1
```

**Review by Nana-EC — DISMISSED** [review]

**Review by Nana-EC — APPROVED** [review]

**Pull request merged**